.github/workflows/github-actions-demo.yml

```diff
@@ -8,7 +8,7 @@ jobs:
         uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
-          go-version: '1.16.7'
+          go-version: '1.17.2'
```

Comment:
why are we deviating from the version used elsewhere?

Reply:
The auto-lint kept failing with an odd error: https://github.com/SkynetLabs/skynet-accounts/runs/4458893545?check_suite_focus=true

I still don't know why that's happening, only that switching to a newer go version fixes it.